### PR TITLE
Fixed Issue #715

### PR DESCRIPTION
**Pull request by aryan-02**
I have fixed issue #715 
The horizontal alignment of both the content sections and buttons is fixed by editing the markup and the custom CSS.

I preferred adding custom CSS instead of using BootStrap, and it works pretty well :)

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
please provide a  demo link and squash your commits

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@aryan-02  Please provide a preview link

**Comment by aryan-02** [comment]
Here's the preview link: https://aryan-02.github.io/fossasia_site_edit/

File: _config.yml [hunk 1]
```diff
@@ -1 +1 @@
-baseurl: ""
\ No newline at end of file
+baseurl: ""
```
Comment (abishekvashok) [inline _config.yml:1]: is this required?

Comment (aryan-02) [reply _config.yml:1]: Actually it got edited accidentally and then I changed it back to its original state. I guess I mistakenly let this line remain. So this is not required. Should I just remove that quickly?

Comment (aryan-02) [reply _config.yml:1]: Changed that

Comment (aryan-02) [reply _config.yml:1]: Oh, sorry.
That line (baseurl: "") is a part of the YML config file in the main repository. So I guess it should remain.

**Comment by aryan-02** [comment]
Okay, it seems @samyakbvs has already fixed it. Should I close the pull request?
I'm so sorry I didn't notice before...

**Pull request closed without merging**